### PR TITLE
Rename p tag to para to prevent method conflict

### PR DESCRIPTION
**Pull request by BlakeWilliams**
https://github.com/luckyframework/web/issues/128

### CODE REVIEW

**Comment by BlakeWilliams** [comment]
I went ahead and followed the same pattern as `TAGS`. My first pass didn't but this ended up being more clear since it follows the existing pattern and also sets us up for more renamed tags if necessary.

File: src/lucky_web/tags/base_tags.cr [hunk 1]
```diff
@@ -35,7 +36,11 @@ module LuckyWeb::BaseTags
   end
 
   {% for tag in TAGS %}
-    generate_tag_methods({{tag.id}})
+    generate_tag_methods({{tag.id}}, {{tag.id}})
+  {% end %}
+
+  {% for name, tag in RENAMED_TAGS %}
+    generate_tag_methods({{name}}, {{tag}})
```
Comment (paulcsmith) [inline src/lucky_web/tags/base_tags.cr]: Could you use named args here and above so it's a bit more clear

```crystal
generate_tag_methods(method_name: {{ name }}, tag: {{ tag }})
```

Comment (BlakeWilliams) [reply src/lucky_web/tags/base_tags.cr]: That's a lot better, good call!

**Comment by paulcsmith** [comment]
Thanks man!

**Pull request merged**